### ISSUE TITLE
[docs] scons fails with master on macOS

### ISSUE THREAD

**Issue by virtualritz** [godotengine/godot#22195]
I'm trying to build master following the instructions [here](http://docs.godotengine.org/en/3.0/development/compiling/compiling_for_osx.html):

```
$ scons platform=osx
scons: Reading SConscript files ...
TypeError: can_build() takes exactly 2 arguments (1 given):
  File "/Users/moritz/code/godot/SConstruct", line 399:
    if (config.can_build(selected_platform)):
```

So it seems scons now needs some additional argument to run that is not documented (yet).

**Comment by neikeq** [comment]
`can_build` was changed to receive two parameter instead of one. The change was introduced in 3334209a732737274a8598191267573fcb989a99.
From what I see in the error message, your SConscript doesn't have those changes, so this seems to be an older version. However, the module that is being built has a `can_build` function that expects two parameters.
I would either try to downgrade the module to a point in history that uses the old `can_build` signature, or upgrade Godot to latest.

**Comment by neikeq** [comment]
Actually, I just went to the list of `config.py` files and it seems that the ARVR module is using the old `can_build` signature: https://github.com/godotengine/godot/blob/a4e4dd396665a29dac63339930a97852bb8cd449/modules/gdnative/arvr/config.py#L1

Not sure if that's the could be the cause you are getting the error. I would build Godot with `module_arvr_enabled=no` to confirm.

cc @akien-mga

**Comment by virtualritz** [comment]
You mean:
```
scons platform=osx module_arvr_enabled=no
```

Same result:
```
scons: Reading SConscript files ...
TypeError: can_build() takes exactly 2 arguments (1 given):
  File "/Users/moritz/code/godot/SConstruct", line 399:
    if (config.can_build(selected_platform)):
```

**Comment by akien-mga** [comment]
Issue template asked for the Godot commit, that would be very relevant information here...

You're likely tracking an outdated repository/branch, `SConstruct:399` is no longer the line shown in your backtrace. There's a try/error branch now that supports both 1-arg and 2-arg `can_build`.

**Comment by akien-mga** [comment]
> Actually, I just went to the list of config.py files and it seems that the ARVR module is using the old can_build signature: 

Fixed in f148e8eed.
